.github/workflows/test.yml

```diff
@@ -16,6 +16,11 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    services:
```

Comment:
Like we did in SAAS to run integration tests on CI.

Reply:
Are you sure we need this? In the SaaS repo for example I'm building container images without dind which means that the docker daemon is present